packages/dd-trace/test/appsec/waf/index.spec.js

```diff
@@ -7,6 +7,14 @@ const Reporter = require('../../../src/appsec/reporter')
 const web = require('../../../src/plugins/util/web')
 
 describe('WAF Manager', () => {
+  const knownAddresses = new Set([
```

Comment:
Can we construct the Set with the constants from `packages/dd-trace/src/appsec/addresses.js` as it has been done in [packages/dd-trace/test/appsec/waf/waf_context_wrapper.spec.js](https://github.com/DataDog/dd-trace-js/pull/4656/files#diff-ac9108ef47852a2c1fffcd2f56c974b106a219bcaf1419bff66e7a2aaf889eccR8)?

Reply:
I didn't want to modify the tests, and the tests in this file are using address strings directly, without using constants from `addresses.js`. The file `waf_context_wrapper.spec.js` is using the constants, so the Set is also using the constants.